pkg/config/config.go

```diff
@@ -147,6 +147,22 @@ type Homebrew struct {
 	Goarm                 string               `yaml:"goarm,omitempty"`
 }
 
+// Krew contains the krew section.
+type Krew struct {
+	IDs                   []string     `yaml:"ids,omitempty"`
+	Name                  string       `yaml:"name,omitempty"`
+	Index                 RepoRef      `yaml:"index,omitempty"`
+	CommitAuthor          CommitAuthor `yaml:"commit_author,omitempty"`
+	CommitMessageTemplate string       `yaml:"commit_msg_template,omitempty"`
+	Caveats               string       `yaml:"caveats,omitempty"`
```

Comment:
(ditto same comments as Description below)

I mostly say these because you might need to carry over the YAML node as-is (with indentation and which YAML multiline mode is used) to the final Krew YAML rendering. Similarly, if there are structural differences such as whitespaces between the old YAML vs the new YAML, then these PRs need to get manually reviewed (krew-release-bot won't be able to approve)

Reply:
will note all those in the docs...